client/src/NotFound.re

```diff
@@ -1,22 +1,30 @@
+[@bs.config {jsx: 3}];
+
 open Utils;
 
 Modules.require("./NotFound.css");
 
-let component = ReasonReact.statelessComponent("NotFound");
+let make = (~entity="path") => {
+  <div className="NotFound">
+    <main>
+      <section>
+        <span> "404"->str </span>
+        <p>
+          "The requested "->str
+          <strong> entity->str </strong>
+          " could not be found"->str
+        </p>
+      </section>
+    </main>
+  </div>;
+};
+
+module Jsx2 = {
+  let component = ReasonReact.statelessComponent(__MODULE__);
+
+  let make = children => {
+    let children = React.array(children);
```

Comment:
@maxlibin I see originally there are no children in component
`let make = (~entity="path", _children) => {`
Why you add children for JSX3 version? @jchavarri Sorry, am i correct here? I need to know too :-) Thanks!

Reply:
you mean adding to jsx 2, I might be wrong but I assume we always need pass in `_children` in Jsx 2 even is not used.

Reply:
As discussed, let's try to migrate all components at once, so there is no need to write any `Jsx2`, which is super confusing. I started a PR with bsconfig configured to use jsx3 in #321, and fixed already a few components.

@max @andrii-antiukhov To make the code easier to review, you can open PRs against the branch in #321, e.g. https://github.com/Sketch-sh/sketch-sh/compare/jsx3...maxlibin:feature/jsx3?expand=1

Thanks!

Reply:
> To make the code easier to review

There is no need to finish a folder or anything before opening a PR. I largely prefer multiple small PRs often, than one huge PR, so reviewing is easier, and also we avoid 2 people doing the same work twice :)